backend/tests/prisonerPersonal.test.ts

```diff
@@ -1,6 +1,10 @@
 import prisonerPersonal from '../controllers/prisonerProfile/prisonerPersonal'
 import config from '../config'
-import { curiousApi } from '../apis'
```

Comment:
I don't see any tests for the new agency checks?

This applies to prisonerProfile tests as well.

Reply:
As discussed - add the app config to the top of the file